src/CompanyService.Data/DepartmentUserRepository.cs

```diff
@@ -20,14 +21,14 @@ public class DepartmentUserRepository : IDepartmentUserRepository
       _provider = provider;
     }
 
-    public bool Add(DbDepartmentUser departmentUser)
+    public bool Add(List<DbDepartmentUser> departmentUsers)
```

Comment:
why do you remove old methods? Now we have to create extra collenctions in each place were this were used. We talk about using in collections in broker because we wanted to reduce their number. do we do this for the repository?

Reply:
added new methods